Wake On Lan/Modules/AboutScreen/Sources/AboutScreen/AboutScreenViewViewModel.swift

```diff
@@ -4,9 +4,9 @@
 //
 //  Created by Dmitry on 08.08.2021.
 //
-
 struct AboutScreenViewViewModel {
     /// Represents header view
+    /// sourcery: AutoEquatable
```

Comment:
AutoEquatable is not required here

Reply:
Removed. I noticed that I already was making it AutoEquatable in AboutScreenHeaderViewViewModel. Thank you.